sdk/src/transaction/versioned/sanitized.rs

```diff
@@ -33,6 +33,10 @@ impl SanitizedVersionedTransaction {
         &self.message
     }
 
+    pub fn get_signatures(&self) -> &Vec<Signature> {
+        &self.signatures
+    }
+
```

Comment:
This function doesn't look to be used anywhere

Reply:
done